examples/components/progress/noAnimation/index.html

```diff
@@ -15,6 +15,11 @@
         <script type="text/javascript" src="../../../../src/framework/core/js/FluidView.js"></script>
 
         <script type="text/javascript" src="../../../../src/components/progress/js/Progress.js"></script>
+        <style>
+            input[type=range]:focus {
+                outline: 1px solid black;
+            }
+        </style>
```

Comment:
Could just add the fl-focus class from fluid.css to the element or one of it's parents instead.